pep-0689.rst

```diff
@@ -1,15 +1,24 @@
 PEP: 689
 Title: Unstable C API tier
 Author: Petr Viktorin <encukou@gmail.com>
-Status: Draft
+Status: Deferred
 Type: Standards Track
 Content-Type: text/x-rst
 Requires: 523
 Created: 22-Apr-2022
-Python-Version: 3.11
+Python-Version: 3.12
 Post-History: `27-Apr-2022 <https://mail.python.org/archives/list/python-dev@python.org/thread/PQXSP7E2B6KNXTJ2AERWMKKX42YP5D6O/>`__,
 
 
+Deferral note
+===============
+
+This PR was accepted provided that function names in the unstable API start
```

Comment:
Did you mean 'this PEP' rather than 'this PR'?

A

Reply:
I did. Thanks for the catch!

Reply:
Fix in https://github.com/python/peps/pull/2771